BlogAPI.Persistence/EntityFrameworkCore/BlogDbContext.cs

```diff
@@ -0,0 +1,22 @@
+﻿using BlogAPI.Persistence.Entities;
+using Microsoft.EntityFrameworkCore;
+using System.Reflection;
+
+namespace BlogAPI.Persistence
+{
+    public class BlogDbContext : DbContext
+    {
+        public BlogDbContext (DbContextOptions<BlogDbContext> options) :
+            base (options)
+        {
```

Comment:
Formatting here seems a bit awkward. Do you have some sort of a linter setup?

Reply:
I'll go with the default .editorconfig and configure to run code cleanup on save